debian/control

```diff
@@ -4,7 +4,7 @@ Priority: optional
 Maintainer: Gecos Team <gecos@guadalinex.org>
 Build-Depends: debhelper (>= 8), cdbs (>=0.4.43), python-all-dev (>=2.3.5-11), python-support, git
 Build-Depends-Indep: gettext, python-distutils-extra
-Standards-Version: 3.8.3
+Standards-Version: 2.1.4-1gecos1
```

Comment:
"Standards-Version" is not related to the version of this package.

https://people.canonical.com/~cjwatson/ubuntu-policy/policy.html/ch-controlfields.html#s-f-Standards-Version

Reply:
Done.